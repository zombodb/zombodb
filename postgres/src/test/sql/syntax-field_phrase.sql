SELECT assert(count(*), 4, 'syntax-field_phrase') FROM so_posts WHERE zdb(so_posts) ==> 'title:"function overloading"';
