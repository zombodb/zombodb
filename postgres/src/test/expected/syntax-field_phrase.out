SELECT assert(count(*), 4, 'syntax-field_phrase') FROM so_posts WHERE zdb(so_posts) ==> 'title:"function overloading"';
 assert 
--------
 t
(1 row)

